fusor-ember-cli/app/controllers/engine/discovered-host.js

```diff
@@ -55,7 +55,7 @@ export default Ember.Controller.extend({
     } else {
       return availableHosts;
     }
-  }.property('availableHosts.[]', 'searchString', 'model', 'isStarted'),
+  }.property('availableHosts.[]', 'searchString', 'isStarted'),
```

Comment:
@jwmatthews, this was the culprit of the Hypervisor tab not enabling for some strange reason.